perses/app/setup_relative_calculation.py

```diff
@@ -51,6 +52,9 @@ def getSetupOptions(filename):
     if 'protocol-type' not in setup_options:
         setup_options['protocol-type'] = 'default'
 
+    if 'run_type' not in setup_options:
```

Comment:
this is specific to `neq`, since often we do not want to run equilibration followed by annealing, but instead resample from a subset of decorrelated equilibrium snapshots that are saved to disk